Source/Views/TextViews/SPTextView.m

```diff
@@ -1910,7 +1910,19 @@ - (void)insertAsSnippet:(NSString*)theSnippet atRange:(NSRange)targetRange
 
 		// Registering for undo
 		[self breakUndoCoalescing];
-		[self.textStorage appendAttributedString:[[NSAttributedString alloc] initWithString:snip]];
+
+        NSMutableAttributedString *tmpAttStr = [[NSMutableAttributedString alloc] initWithString:snip];
+
+        // if we are inserting a favourite query,
+        // add the font
+        // FIXME: should this apply to all snippet inserts actually?
```

Comment:
Not sure

Reply:
I think this should apply to all snipped inserts, yes?